test/scaling/nointerp.jl

```diff
@@ -20,6 +20,38 @@ using Interpolations, Test, LinearAlgebra, Random
 
     @test length(Interpolations.gradient(sitp, pi/3, 2)) == 1
 
+    xs = range(-pi, stop=pi, length=60)
+    A = hcat(map(f1, xs), map(f2, xs), map(f3, xs))
+    itp = interpolate(A, (BSpline(Cubic(Periodic(OnGrid()))), NoInterp()))
+    sitp = scale(itp, xs, ys)
+
+    for x in (xs[6:end-5] .+ 0.05), y in ys
+        if y in (1,2)
+            @test_broken h = @inferred(Interpolations.hessian(sitp, x, y))
+            h = Interpolations.hessian(sitp, x, y)
+            @test ≈(h[1], -f(x,y), atol=0.05)
```

Comment:
This is fine as-is, but just FYI you can also write `@test h[1] ≈ -f(x,y) atol=0.05` if you like that better. (This file probably uses the "call" form for historical reasons, from before that syntax was possible, and was updated by search/replace.)

Reply:
Yes, I'm familiar with this syntax, but didn't want to change style halfway. I have changed it now to this "modern" form for better readability in the test files related to this PR.